khwilo/questioner
# #163972787 Enable a user to sign in

#### What does this PR do?

Allow a user to log in to the Questioner website.

#### Description of Task to be completed?

- [x] Consume the REST API endpoint 
`POST https://q-questioner-api.herokuapp.com/api/v2/auth/login` when logging in a user.

#### How should this be manually tested?

1.  Clone the repository 
2. Open the file **signin.html** on your web browser.
3. Try to log in a user

**NOTE**: A user should be registered first. 
- For an admin user have the _username_ to be **watai** and _password_ to be **questioner_1234**.
- For a regular user have the _username_ to be **stringer** and _password_ to be **john12345**.

#### What are the relevant pivotal tracker stories?

[#163972787](https://www.pivotaltracker.com/story/show/163972787)

#### Screenshots

1. Admin user signing in:

![admin-user-login](https://user-images.githubusercontent.com/5740429/52853461-c41bf780-312c-11e9-91c7-9a13c9f4d196.gif)

2. Regular user signing in:

![regular-user-login](https://user-images.githubusercontent.com/5740429/52853500-dd24a880-312c-11e9-8029-c0d13dbed25c.gif)
